web-marketplace/src/components/organisms/CreateSellOrderForm/CreateSellOrderForm.tsx

```diff
@@ -55,7 +55,7 @@ const CreateSellOrderForm: React.FC<Props> = ({
     price: undefined,
     askDenom: undefined,
     amount: undefined,
-    enableAutoRetire: true,
+    enableAutoRetire: false,
```

Comment:
This is the main fix.
The rest of the changes are about fixing the sell orders extended query to fetch *all* sell orders and not only the first 100 ones (that the ledger returns by default) so the fix can be tested with redwood on dev and deploy previews (redwood has more than 100 active sell orders so new sell orders weren't being displayed)